test/Microsoft.DocAsCode.MarkdownRewriters.Tests/MarkdigMarkdownRewritersTests.cs

```diff
@@ -103,6 +103,20 @@ public void TestMarkdigMarkdownRewriters_MailTo()
             var source = "<Mailto:docs@microsoft.com>";
             var expected = "<docs@microsoft.com>\n\n";
 
+            var result = Rewrite(source, "topic.md");
+            Assert.Equal(expected, result);
+
+            result = Rewrite(result, "topic.md");
+            Assert.Equal(expected, result);
+        }
+
+        [Fact]
+        [Trait("Related", "MarkdigMarkdownRewriters")]
+        public void TestMarkdigMarkdownRewriters_MailTo2()
+        {
+            var source = "<Mailto:docs@microsoft.com>";
+            var expected = "<docs@microsoft.com>\n\n";
+
             var result = Rewrite(source, "topic.md");
             Assert.Equal(expected.Replace("\r\n", "\n"), result);
```

Comment:
Does the above case already cover this one?

Reply:
nice catch!